source/k64f_emac.c

```diff
@@ -583,7 +583,7 @@ static err_t k64f_low_level_output(struct netif *netif, struct pbuf *p)
       break;
   if (q != NULL) {
     // Allocate properly aligned buffer
-    psend = (uint8_t*)malloc(p->tot_len);
+    psend = (uint8_t*)pbuf_alloc(PBUF_RAW,p->tot_len,PBUF_POOL);
```

Comment:
It's 8, which in turn is a requirement of the K64F DMA:

https://github.com/ARMmbed/sal-driver-lwip-k64f-eth/blob/7bde6e31979b750b16ecfcb5e74cb5fb0094425f/sal-driver-lwip-k64f-eth/k64f_emac_config.h#L41

The alignment of `malloc` is already 8, so that wasn't an issue with malloc, but this might need to change. An example of how to handle for RX buffers (which need to be aligned to 16 bytes):

https://github.com/ARMmbed/sal-driver-lwip-k64f-eth/blob/7bde6e31979b750b16ecfcb5e74cb5fb0094425f/source/k64f_emac.c#L184